pystan/experimental/pickling_tools.py

```diff
@@ -32,6 +32,11 @@ def create_fake_model(module_name):
     StanModel
         Dummy StanModel object with specific module name.
     """
+    tbb_dir = os.getenv("STAN_TBB")
```

Comment:
I feel like we should be consistent here. Either we allow all libraries to set their paths with environment variables (my preference), or we let none of them do so. And even if we introduce this new practice, we should do it in a different pull request.

Reply:
True, I forgot to fix this code. It should not need STAN_TBB call.